portfoliostyle.css

```diff
@@ -0,0 +1,438 @@
+
+
+* {
+	box-sizing: border-box;
+	font-family: 'Nunito', sans-serif;
+}
+
+section, footer div {
+	width: 65%;
+	margin: 0 auto;
+}
+
+body {
+	background-color: #D3D3D3;
+	background-image: linear-gradient(#808080, #FFFFFF);
+	background-position: top center;
+}
+
+/*most element styles*/
+
+nav {
```

Comment:
maybe change the text in the nav to have more contrast for accessibility, especially after the media query.

Reply:
Changed to have better contrast